README.md

```diff
@@ -249,94 +100,61 @@ jobs:
+| `config-file`         | A path to a file in the current repository or an external repository. Use this syntax for external files: `OWNER/REPOSITORY/FILENAME@BRANCH`                                             | local file: `./.github/dependency-review-config.yml`, <br> external repo: `github/octorepo/dependency-review-config.yml@main` |
+| `external-repo-token` | Specifies a token for fetching the configuration file, if the file resides in a private external repository. Create a token in [developer settings](https://github.com/settings/tokens). | Any token with `read` permissions to the repository hosting the config file.                                                   |
 
-```yaml
-- name: Dependency Review
-  uses: actions/dependency-review-action@v2
-  with:
-    fail-on-severity: high
-```
+#### Example
 
-### Dependency Scoping
-
-By default the action will only fail on `runtime` dependencies that have vulnerabilities or unacceptable licenses, ignoring `development` dependencies. You can override this behavior with the `fail-on-scopes` option, which will allow you to list the specific dependency scopes you care about. The possible values are: `unknown`, `runtime`, and `development`. Note: Filtering by scope will not be supported on Enterprise Server just yet, as the REST API's introduction of `scope` will be released in an upcoming Enterprise Server version. We will treat all dependencies on Enterprise Server as having a `runtime` scope and thus will not be filtered away.
+Start by specifying that you will be using an external configuration file:
 
 ```yaml
 - name: Dependency Review
   uses: actions/dependency-review-action@v2
   with:
-    fail-on-scopes: runtime, development
+    config-file: './.github/dependency-review-config.yml'
 ```
 
-### Licenses
-
-You can set the action to fail on pull requests based on the licenses of the dependencies
-they introduce. With `allow-licenses` you can define the list of licenses
-your repository will accept. Alternatively, you can use `deny-licenses` to only
-forbid a subset of licenses. These options are not supported on Enterprise Server.
-
-You can use the [Licenses
-API](https://docs.github.com/en/rest/licenses) to see the full list of
-supported licenses. Use [SPDX licenses](https://spdx.org/licenses/)
-to filter the licenses. A couple of examples:
-
-```yaml
-# only allow MIT-licensed dependents
-- name: Dependency Review
-  uses: actions/dependency-review-action@v2
-  with:
-    allow-licenses: MIT
-```
+And then create the file in the path you just specified: 
 
 ```yaml
-# Block Apache 1.1 and 2.0 licensed dependents
-- name: Dependency Review
-  uses: actions/dependency-review-action@v2
-  with:
-    deny-licenses: Apache-1.1+
+fail-on-severity: 'critical'
+allow-licenses:
+  - 'GPL-3.0'
+  - 'BSD-3-Clause'
+  - 'MIT'
```

Comment:
Is having quotes right here? In the current README the individual YAML examples don't have quotes 🤔

Reply:
Quotes in YAML are optional, can be taken out if needed!